lib/beaker-hostgenerator/data.rb

```diff
@@ -685,6 +685,22 @@ def osinfo
             'platform' => 'fedora-14-i386',
           },
         },
+        'freebsd12-64' => {
+          :general => {
+            'platform' => 'freebsd-12-amd64',
```

Comment:
I don't exactly understand the "platform" string. `amd64` is FreeBSD way to say x86_64 (like Debian), but in other places I've seen this still being used as `freebsd-123-x86_64`, so no idea…

Reply:
I always struggle with this myself and need to dig through the code or wait for the awesome ewoud...

Reply:
the failures in https://github.com/voxpupuli/puppet-mosquitto/actions/runs/4269959304/jobs/7433482798 used `freebsd-123-amd64`, so I am going to stick with that for now.

Reply:
also https://github.com/voxpupuli/rspec-puppet-facts/blob/1418bca958049060a77dd49f10a9a298d15dc438/lib/rspec-puppet-facts.rb#L71-L72